lib/ex_aws/request.ex

```diff
@@ -83,7 +85,11 @@ defmodule ExAws.Request do
           )
 
         {:error, %{reason: reason}} ->
-          Logger.warn("ExAws: HTTP ERROR: #{inspect(reason)}")
+          Logger.warn(
+            "ExAws: HTTP ERROR: #{inspect(reason)} for URL: #{inspect(safe_url)} HEADERS: #{
+              inspect(full_headers)
+            } BODY: #{inspect(req_body)} ATTEMPT: #{attempt}"
```

Comment:
Inspecting the body can be problematic here as it can be arbitrarily large. We can maybe print the byte size of the body, but `inspect(req_body)` is guaranteed to cause major issues.

Reply:
@benwilson512 Thanks! You're right. I also thought about request headers which can contain sensitive information (e.g. Authorization header). So I just removed both contents from logging. bd058c0